.github/workflows/verify_consumer_pacts.yml

```diff
@@ -319,7 +319,7 @@ jobs:
                           -e JANITOR_CLIENT_CREDENTIAL_FILE_PATH="" \
                           -e JANITOR_TRACK_RESOURCE_PROJECT_ID="" \
                           -e JANITOR_TRACK_RESOURCE_TOPIC_ID="" \
-                          sbtscala/scala-sbt:openjdk-17.0.2_1.7.2_2.13.10 \
+                          sbtscala/scala-sbt:eclipse-temurin-jammy-17.0.10_7_1.10.2_2.13.15 \
```

Comment:
here and elsewhere, set the `scala-sbt` docker image to the version that uses sbt 1.10.2 and Scala 2.13.15.